.github/workflows/e2e.yaml

```
@@ -83,14 +83,18 @@ jobs:
 
       - name: Build agent-enclave bundle
         run: |
+          sudo apt-get -y install jq
           mkdir $PAYLOAD_ARTIFACTS
-          ./gen_bundle.sh agent-enclave-bundle:build
+          docker build ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc -f ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc/tools/packaging/build/agent-enclave-bundle/Dockerfile --build-arg SGX_MODE=${SGX_MODE} -t agent-instance:build
+          jq -a -f sgx-mode-config.filter config.json.template | sudo tee ${PAYLOAD_ARTIFACTS}/config.json
+          docker export $(docker create agent-instance:build) | sudo tee > ${PAYLOAD_ARTIFACTS}/agent-instance.tar
         working-directory: ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc/tools/packaging/build/agent-enclave-bundle
 
       - name: Build boot-instance bundle
         run: |
-          ./gen_bundle.sh boot-instance-bundle:build
-        working-directory: ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc/tools/packaging/build/boot-instance-bundle
+          docker build ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc -f ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc/tools/packaging/build/boot-instance-bundle/Dockerfile --build-arg SGX_MODE=${SGX_MODE} -t boot-instance:build
+          docker export $(docker create boot-instance:build) | sudo tee > ${PAYLOAD_ARTIFACTS}/boot-instance.tar
+        working-directory: ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc
```

maybe simplify (ditto for `agent-enclave-bundle`)
```suggestion
          docker build . -f tools/packaging/build/boot-instance-bundle/Dockerfile --build-arg SGX_MODE=${SGX_MODE} -t boot-instance:build
          docker export $(docker create boot-instance:build) | sudo tee > ${PAYLOAD_ARTIFACTS}/boot-instance.tar
        working-directory: ${{ github.workspace }}/src/github.com/confidential-containers/enclave-cc
```

---

Yes, boot-instance could, since the build context is the enclave-cc rootdir, but the agent-instance could not be ditto, because there is a jq command need to be executed under agent-enclave-bundle directory, so the working-directory differs between these two bundles.